seed_data/utils_test.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for utils file"""
+import datetime
+import pytz
+from django.test import TestCase
+
+from seed_data.utils import add_year
+
+
+class AddYearTests(TestCase):
+    """Tests for add year method"""
+
+    def test_add_year(self):  # pylint: disable=no-self-use
+        """assert add year method works"""
+        now = datetime.datetime.now(tz=pytz.UTC)
```

Comment:
I would be *very* careful with this, if there's a bug in `add_year` (like the two cases I pointed out above) this test could fail during certain times of certain years. I think you should prefer explicit testing of specific dates that do/don't include a leap day between that date and +1 year.